File: src/main/java/com/baulsupp/oksocial/services/imgur/ImgurUtil.java
package com.baulsupp.oksocial.services.imgur;

import com.google.common.collect.Sets;
import java.util.Collections;
import java.util.Set;
import okhttp3.Request;

public class ImgurUtil {
  private ImgurUtil() {
  }

  public static final Set<String> API_HOSTS =
      Collections.unmodifiableSet(Sets.newHashSet(
          "api.imgur.com")
      );

  public static Request apiRequest(String s, Request.Builder requestBuilder) {
    return requestBuilder.url("https://api.imgur.com" + s).build();
  }
}

File: src/main/kotlin/com/baulsupp/oksocial/services/imgur/ImgurUtil.kt
package com.baulsupp.oksocial.services.imgur

import okhttp3.Request

object ImgurUtil {

  val API_HOSTS = setOf((
      "api.imgur.com")
  )

  fun apiRequest(s: String, requestBuilder: Request.Builder): Request {
    return requestBuilder.url("https://api.imgur.com" + s).build()
  }
}
